### PR TITLE
Commands: Make API Review changes

### LINKED ISSUE #1774: Commands: Migration tab expansion should show names (not ids)
Tab expansion on migrations currently shows IDs (like *20150306115213_InitialCreate*). Instead, we should show (and complete) names (like *InitialCreate*) since they'll be used most.

### PR DESCRIPTION
**Pull request by bricelam**
Resolves #1774

No functional changes here.

### CODE REVIEW

File: src/EntityFramework.Commands/Commands/Program.cs [hunk 1]
```diff
@@ -10,67 +10,84 @@
...
-            Console.CancelKeyPress += (sender, eventArgs) =>
-            {
-                _applicationShutdown.RequestShutdown();
-            };
+            Console.CancelKeyPress += (_, __) => _applicationShutdown.RequestShutdown();
 
             var app = new CommandLineApplication
             {
-                Name = "ef",
+                Name = "dnx ef",
```
Comment (lajones) [inline src/EntityFramework.Commands/Commands/Program.cs]: Does this work? I thought you had to declare a Command within a Command to get sub-commands?

File: test/EntityFramework.Commands.FunctionalTests/TestUtilities/OperationExecutorWrapper.cs [hunk 2]
```diff
@@ -55,59 +56,72 @@ public string GetContextType(string name)
...
+            string outputDir,
+            bool fluentOnly)
+            => InvokeOperation<IEnumerable<string>>(
+                "ReverseEngineer",
+                new Hashtable
+                {
+                    { "connectionString", connectionString },
+                    { "provider", provider },
+                    { "outputDir", outputDir },
+                    { "fluentOnly", fluentOnly }
```
Comment (lajones) [inline test/EntityFramework.Commands.FunctionalTests/TestUtilities/OperationExecutorWrapper.cs]: I think the key has to be "useFluentApiOnly" to match the .psm1 file?

**Comment by lajones** [comment]
Assuming above comments are addressed :shipit:

File: src/EntityFramework.Commands/Commands/Program.cs (hunk 1 above)
Comment (bricelam) [inline src/EntityFramework.Commands/Commands/Program.cs]: This is just for display purposes; it's never used by the parser. The user can register it however they want.
```C#
{
  "commands": {
    "unicorn": "EntityFramework.Commands"
  }
}
```

File: test/EntityFramework.Commands.FunctionalTests/TestUtilities/OperationExecutorWrapper.cs (hunk 2 above)
Comment (bricelam) [inline test/EntityFramework.Commands.FunctionalTests/TestUtilities/OperationExecutorWrapper.cs]: Thanks, I had a few merge conflicts in that area...

**Pull request merged**